dcs/task.py

```diff
@@ -323,6 +323,13 @@ class Expend(Enum):
     Half = "Half"
     All = "All"
 
+    def __eq__(self, other: Any) -> bool:
+        if isinstance(other, str):
+            return self.value == other
+        if isinstance(other, Expend):
```

Comment:
Close enough so meh, non-blocking, but for the future just `return super().__eq__(other)` after doing any custom checks.